web-common/src/components/forms/__stories__/MultiInput.stories.svelte

```
@@ -0,0 +1,39 @@
+<script lang="ts">
+  import { Button } from "@rilldata/web-common/components/button";
+  import MultiInput from "@rilldata/web-common/components/forms/MultiInput.svelte";
+  import { Meta, Story } from "@storybook/addon-svelte-csf";
+  import { createForm } from "svelte-forms-lib";
+  import * as yup from "yup";
+
+  const formState = createForm({
+    initialValues: {
+      emails: [],
+    },
+    validationSchema: yup.object({
+      emails: yup.array().of(
+        yup.object().shape({
+          email: yup.string().email("Invalid email"),
```

See here – we should probably surface this error message in the component